handlers/consumer.js

```diff
@@ -60,7 +60,7 @@ class BaseConsumer {
 
   static async connect(options = {}) {
     try {
-      const queue = (process.env.APP_CLUSTER && process.env.APP_SERVICE) ? `${process.env.APP_CLUSTER}-${process.env.APP_SERVICE}-messages` : `local-${process.pid}-messages`;
+      const queue = (process.env.APP_CLUSTER && process.env.APP_CONSUMER_NAME) ? `${process.env.APP_CLUSTER}-${process.env.APP_CONSUMER_NAME}-messages` : `local-${process.pid}-messages`;
```

Comment:
this is the real change for this ticket

Reply:
Late to the party but lets change this to be MESSAGE_BUS_CONSUMER_GROUP